src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,23 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns true if {@code s} represents a 6-digit number in range (100000, 999999).
+     * e.g. 240001, 240002, ..., <br>
+     * Will return false for any other non-null string input
+     * e.g. empty string, "-1", "0", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
+     * @throws NullPointerException if {@code s} is null.
+     */
+    public static boolean isSixDigitNumber(String s) {
+        requireNonNull(s);
+
+        try {
+            int value = Integer.parseInt(s);
+            return value >= 100000
+                    && value <= 999999 && !s.startsWith("+"); // "+1" is successfully parsed by Integer#parseInt(String)
```

Comment:
Have you considered whether there might be a more optimal approach to writing this section of code? 
For example: `return s.matches("[1-9]\\d{5}");`